rasa/core/test.py

```diff
@@ -302,22 +304,21 @@ async def _generate_trackers(
     agent: "Agent",
     max_stories: Optional[int] = None,
     use_e2e: bool = False,
-) -> List[Any]:
+) -> "TrainingDataGenerator":
```

Comment:
we should rename the function `_generate_trackers` then as it no longer generates the trackers.